esp/scm/ws_dfu.ecm

```diff
@@ -133,6 +133,7 @@ ESPStruct [nil_remove] DFUFileDetail
     [min_ver("1.09")] string UserPermission;
     [min_ver("1.21")] string ContentType;
     [min_ver("1.22")] int64 CompressedFileSize;
+    [min_ver("1.34")] float PercentCompressed;
```

Comment:
You may just want to use a string if you want to accurately represent a 2 decimal point number.